app/jinja_filters.py

```diff
@@ -351,7 +351,15 @@ def __init__(
         self.url = block['link'] + '#' + answer['id']
 
         qa_attribute = answer['id'] + '-edit'
-        self.attributes = dict(**{'data-qa': qa_attribute})
+
+        self.attributes = dict(
+            **{
+                'data-qa': qa_attribute,
+                'data-ga': 'click',
+                'data-ga-category': 'Summary',
+                'data-ga-action': 'Edit click',
+            }
+        )
```

Comment:
No need for the the `qa_attribute` variable or `dict`:

```python
self.attributes = {
    'data-qa': answer['id'] + '-edit',
    'data-ga': 'click',
    'data-ga-category': 'Summary',
    'data-ga-action': 'Edit click',
}
```

Reply:
Done 👍